kvmagent/kvmagent/test/localstorage_testsuite/test_revert_volume_snapshot_group_with_memory_snapshot.py

```
@@ -0,0 +1,237 @@
+            elif idx == 3:
+                self.assertEqual(disk.target.dev_, 'hdc')
+            elif idx == 4:
+                self.assertEqual(disk.target.dev_, 'sdd')
+                self.assertEqual(disk.source.file_, virtio_scsi1_path)
+            elif idx == 5:
+                self.assertEqual(disk.target.dev_, 'sdf')
+                self.assertEqual(disk.source.file_, virtio_scsi2_path)
+
+        # create snapshot install path
+        memorySnapshotUuid = uuidhelper.uuid()
+        memorySnapshotPath = os.path.join(env.SNAPSHOT_DIR, '%s.qcow2' % memorySnapshotUuid)
```

Ruff (0.12.2)

`os` may be undefined, or defined from star imports

(F405)